kits/arm/arm.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+import hebi
+from time import sleep, time, perf_counter
+import numpy as np
+
+import os
+import sys
+
+# ------------------------------------------------------------------------------
+# Add the root folder of the repository to the search path for modules
+root_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
+sys.path = [root_path] + sys.path
+# ------------------------------------------------------------------------------
+
+from util.math_utils import get_grav_comp_efforts
+from util.math_utils import gravity_from_quaternion
+
+class ArmParams(object):
+    
+    def __init__(self, family, moduleNames, hrdf):
+        self.family = family
+        self.hrdf = hrdf
+        self.moduleNames = moduleNames
+
+
+    
+class Arm():
```

Comment:
Inherit from `object` like above to be consistent. However, I think this is done automatically for Python 3, and since we don't support Python 2 anymore, we can probably just get rid of this explicit inheritance...